src/llm/generation_chains/__init__.py

```diff
@@ -0,0 +1,6 @@
+from llm.generation_chains.killer_chain import KillerChain
+from llm.generation_chains.rooms_chain import RoomsChain
```

Comment:
Čo toto znamená? Len pre moju zaujímavosť. Že už to potom nemusíš nikde importovať?

Reply:
`__init__.py` subory maju specialne postavenie v adresaroch. hovoria, ze sa mas na ten adresar pozerat ako na pythoni balicek. takze napriklad `llm.generation_chains` je balicek.

ked niekde importujem vela veci z balicku, napriklad tam importujem vsetky chains, tak sa zvykne robit to, ze v `__init__.py` ich takto naimportujes a potom nemusis robit 2 importy

```
from llm.generation_chains.killer_chain import KillerChain
from llm.generation_chains.rooms_chain import RoomsChain
```

ale staci jeden, priamo z toho balicku

```
from llm.generation_chains import (
    KillerChain,
    RoomsChain
)
```